integration/README.md

```diff
@@ -5,23 +5,23 @@ Integration tests that run the real deal and make sure every component works tog
 
 ## Running
 
-First, you'll need to compile Gateway (in release mode) and compile the Ethereum contracts.
+You can run the tests with Gateway compiled in release or debug profile. The default profile is **debug**.
```

Comment:
This is not true, at least not anymore. Really now we should use `yarn build` and `yarn test` which will do all of the right things all the time.